CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Packaging
 
 - Minimum Rust version has been bumped to 1.37.0
+- Added ability to pick Linux/BSD backends(X11/Wayland) during build, both are enabled by default
```

Comment:
```suggestion
- Added Rust features `x11` and `wayland` to pick backends, which are both enabled by default
```

Reply:
`pick backends` sounds kinda weird to me -- maybe `choose backends`?

Also `which are both enabled by default` -> `with both enabled by default`, IMHO.

Reply:
I've decided to avoid `choose` since I read such thing like 'one over another'.